src/udiskslinuxfilesystem.c

```diff
@@ -1701,6 +1737,8 @@ handle_unmount (UDisksFilesystem      *filesystem,
   gboolean fstab_mounted;
   gboolean success;
   UDisksBaseJob *job = NULL;
+  UDisksObject *filesystem_object = NULL;
+  WaitForFilesystemMountPointsData wait_data;
```

Comment:
Oh, one last thing - have a look at the `csmock` CI tests - this struct needs to be initialized _NULL_ otherwise the `g_free()` down there would hit an uninitialized memory.

Reply:
Nice, I wasn't aware of the csmock tests. Good catch, came from moving the code.